docs/proposals/modifying-output-image.md

```diff
@@ -0,0 +1,147 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: modifying-output-image
+authors:
+  - "@ImJasonH"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-04-14
+last-updated: 2021-04-14
+status: provisional
+---
+
+# Modifying Output Image
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [docs](/docs/)
+
+## Open Questions
+
+1. Should this feature also allow users to _unset_ existing labels/annotations inherited from base images?
+
+## Summary
```

Comment:
Should this EP touch upon defining `labels` and `annotations` in both `Build` and `BuildRun` resources?

Reply:
I don't think so. I think that will likely come up when we design dynamic labels and other values provided at buildrun-time.